apps/site/assets/ts/schedule/components/schedule-finder/UpcomingDepartures.tsx

```diff
@@ -135,30 +135,33 @@ const TableRow = ({
   );
 };
 
+const UpcomingDeparturesHeader = (
+  <div className="schedule-table__upcoming-departures-header">
+    <h3>Upcoming Departures</h3>
+    <span className="schedule-table__live-clock" aria-hidden>
+      <span className="icon-realtime animate notranslate">
+        <span
+          className="c-svg__icon-live-clock"
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={{ __html: liveClockSvg }}
+        />
+        <span className="icon-realtime-text">live</span>
+      </span>
+    </span>
+  </div>
+);
+
 export const UpcomingDepartures = ({
   state,
   input
 }: Props): ReactElement<HTMLElement> | null => {
   const { data: journeys, error, isLoading } = state;
 
   if (error || isLoading) return null;
-
-  if (journeys !== null && hasPredictions(journeys)) {
-    return (
-      <>
-        <div className="schedule-table__upcoming-departures-header">
-          <h3>Upcoming Departures</h3>
-          <span className="schedule-table__live-clock" aria-hidden>
-            <span className="icon-realtime animate notranslate">
-              <span
-                className="c-svg__icon-live-clock"
-                // eslint-disable-next-line react/no-danger
-                dangerouslySetInnerHTML={{ __html: liveClockSvg }}
-              />
-              <span className="icon-realtime-text">live</span>
-            </span>
-          </span>
-        </div>
+  return (
+    <>
+      {UpcomingDeparturesHeader}
+      {journeys !== null && hasPredictions(journeys) ? (
```

Comment:
Is `UpcomingDeparturesHeader` supposed to be always shown or only when `journeys !== null && hasPredictions(journeys)` as in the previous version of the file? i.e. should lines 163 and 164 be swapped?

Reply:
The design called for it being always visible!

Reply:
Disregard then 😅